templates/docs/examples/patterns/segmented-control.html

```diff
@@ -0,0 +1,72 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Segmented Control{% endblock %}
+
+{% block standalone_css %}patterns_segmented-control{% endblock %}
+
+{% block content %}
+
+<div class="p-segmented-control">
+  <button>One</button>
+  <button><i class="p-icon--information"></i> Two</button>
+  <button>Three</button>
+  <a class="p-button" href="#">Link</a>
+</div>
+
+<br><br>
+
+<div class="p-segmented-control">
+  <label class="p-segmented-control__radio">
+    <input type="radio" name="radio" aria-labelledby="radio-one-input">
```

Comment:
Do we need both `button` and `radio` implementations? If so, what are the usecases for them? Documentation doesn't mention it, so which one should developer choose?

Reply:
Existing implementations (Charmhub etc) are just buttons, not radios, so we went with buttons at first, and thought a wrapper around existing button pattern would be easy to understand.

However, the designs show a selected state, which we don't need in the button use case, but we could see that this pattern could easily be used as an alternative to content tabs, so we wanted to show we could support both. If we had to drop one, I'd think it would be radios since it hasn't been created in other projects already - what do you think?